### DISCUSSION TITLE
Is there any way to open two different windows at the same time, and then have them both open a dialog that shares the same context?

### DISCUSSION THREAD

**Discussion by nir34** [AvaloniaUtils/DialogHost.Avalonia#42] [Q&A] [answered]
it seems that a dialog  cannot be owned by more than one window

**Comment by SKProCH** [comment, marked as answer]
You can use the same viewmodel for both DialogHost at the same time, so content will be linked and updated through bindings.

But you can't use the same Avalonia control (such as UserControl or any other) for multiple DialogHosts, since Avalonia control can have only one parent control.
